aiida/cmdline/commands/cmd_node.py

```diff
@@ -98,18 +98,18 @@ def _copy_tree(key, output_dir):  # pylint: disable=too-many-branches
         Recursively copy the content at the ``key`` path in the given node to
         the ``output_dir``.
         """
-        for file in node.list_objects(key=key):
+        for file in node.list_objects(key):
             # Not using os.path.join here, because this is the "path"
             # in the AiiDA node, not an actual OS - level path.
             file_key = file.name if not key else key + '/' + file.name
-            if file.type == FileType.DIRECTORY:
+            if file.file_type == FileType.DIRECTORY:
```

Comment:
Renamed `.type` -> `.file_type`.

Reply:
I have a deprecation for this. The only potential problem is that again (like with the `ExitCode`) I have changed `File` from named tuple into a class. There at least I have backported a `type` property that is deprecated and also constructor accepts `type` forwarding it to `file_type`. Hope that is enough.

Reply:
Hmm, yeah I could see someone doing something like 

```python
for name, type in files_list:
    ...
```

Probably more something that would show up in `aiida-core` itself, though.

Reply:
Maybe implementing an `__iter__` method (with deprecation warning) would be nice? Something like 

```python
def __iter__(self):
    <deprecation message>
    yield self.name
    yield self.type
```